druid/src/widget/widget.rs

```diff
@@ -222,6 +223,37 @@ pub trait Widget<T> {
             ..Default::default()
         }
     }
+
+    /// Computes max intrinsic/preferred dimension of a widget on the provided axis.
```

Comment:
Putting a newline after the first short sentence will make the rustdoc output look better, I think